### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by szpotona**
Please review my code.

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-e git+https://github.com/cloudify-cosmo/cloudify-rest-client@master#egg=cloudify-rest-client==3.1
```
Comment (idanmo) [inline dev-requirements.txt]: Versions inconsistency.

File: blueprint_plugin.yaml.template [hunk 2]
```diff
@@ -0,0 +1,51 @@
+tosca_definitions_version: cloudify_dsl_1_0
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.1rc1/types.yaml
```
Comment (idanmo) [inline blueprint_plugin.yaml.template]: Import types.yaml of 3.1 ?

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,15 @@
+[tox]
+envlist=flake8,py27
```
Comment (idanmo) [inline tox.ini]: Any idea if this plugin should be py26 compatible?

File: dev-requirements.txt (hunk 1 above)
Comment (szpotona) [inline dev-requirements.txt]: Sorry, my bad. Updated.

File: blueprint_plugin.yaml.template (hunk 2 above)
Comment (szpotona) [inline blueprint_plugin.yaml.template]: Updated.

File: tox.ini (hunk 3 above)
Comment (szpotona) [inline tox.ini]: I can add py26 as well, there shouldn't be any features that are not supported.

**Comment by barakm** [comment]
@szpotona, @idanmo , @nir0s what is the status of this PR?

**Comment by szpotona** [comment]
This PR was left to review and merge. I finished development of the functionalities.

**Pull request merged**